core/test_network/src/lib.rs

```diff
@@ -141,13 +138,13 @@ impl TestGenesis for GenesisBlock {
 pub struct NetworkBuilder {
     n_peers: u32,
     port: Option<u16>,
-    config: Option<Config>,
```

Comment:
Is this change related to rebasing?

Reply:
No it is because genesis block hash was added to the `Config`. Only one test set custom config so I added specific method for this test instead (`with_debug_force_soft_fork`)